### PR TITLE
Update migration.md

### PR DESCRIPTION
**Pull request by Wenting0905**
tiny changes and PTAL @shenli

### CODE REVIEW

**Comment by queenypingcap** [comment]
@andelf PTAL.

File: op-guide/migration.md [hunk 1]
```diff
@@ -254,26 +255,43 @@ host = "127.0.0.1"
...
 
-###4. Inserting data into MySQL
+### 4. Inserting data into MySQL
 
 ```bash
 INSERT INTO t1 VALUES (4, 4), (5, 5);
 ```
 
-###5. Login TiDB and view the data:
+### 5. Login TiDB and view the data:
```
Comment (andelf) [inline op-guide/migration.md]: Use same tense as other 4 titles

**Comment by queenypingcap** [comment]
@andelf PTAL.

File: op-guide/migration.md [hunk 2]
```diff
@@ -254,26 +255,43 @@ host = "127.0.0.1"
...
 
-###4. Inserting data into MySQL
+### 4. Inserting data into MySQL
 
 ```bash
 INSERT INTO t1 VALUES (4, 4), (5, 5);
 ```
 
-###5. Login TiDB and view the data:
+### 5. Logging in TiDB and viewingthe data:
```
Comment (andelf) [inline op-guide/migration.md]: Typo, a missing space

Comment (queenypingcap) [inline op-guide/migration.md]: Thanks!

**Comment by andelf** [comment]
LGTM， should resolve conflicts

**Pull request merged**